### DISCUSSION TITLE
Cara membuat sampling dari Populasi data besar

### DISCUSSION THREAD

**Discussion by AnthonySSSSS** [teamalgoritma/community#467] [Q&A] [answered]
Contoh saya memiliki 1jt data, contoh, apakah bisa jika saya ambil 100 data dari 1juta data sudah mewakili sebagai sampel yang dapat dipertanggung jawabkan untuk data yang saya olah ? mohon dibantu referensi bacaan atau mungkin cara terbaik membuat sample data dari data yang cukup besar/banyak.

**Comment by kevbow** [comment, marked as answer]
Hi @AnthonySSSSS ,

Please refer to this [discussion 361](https://github.com/teamalgoritma/community/discussions/361)
